src/content/LandingPage/LandingPage.js

```diff
@@ -1,7 +1,104 @@
+          <Breadcrumb noTrailingSlash aria-label="Page navigation">
+            <BreadcrumbItem>
+              <a href="/">Getting started</a>
+            </BreadcrumbItem>
+          </Breadcrumb>
+          <h1 className="landing-page__heading">
+            Design &amp; build with Carbon
+          </h1>
+        </div>
+      </div>
+      <div className="bx--row landing-page__r2">
+        <div className="bx--col bx--no-gutter">
+          <Tabs {...props.tabs} aria-label="Tab navigation">
+            <Tab {...props.tab} label="About">
+              <div className="bx--grid bx--grid--no-gutter bx--grid--full-width">
+                <div className="bx--row landing-page__tab-content">
+                  <div className="bx--col-md-4 bx--col-lg-7">
+                    <h2 className="landing-page__subheading">
+                      What is Carbon?
+                    </h2>
+                    <p className="landing-page__p">
+                      Carbon is IBM’s open-source design system for digital
+                      products and experiences. With the IBM Design Language as
+                      its foundation, the system consists of working code,
+                      design tools and resources, human interface guidelines,
+                      and a vibrant community of contributors.
+                    </p>
+                    <Button>Learn more</Button>
+                  </div>
+                  <div className="bx--col-md-4 bx--offset-lg-1 bx--col-lg-8">
+                    <img
+                      className="landing-page__illo"
+                      src={`${process.env.PUBLIC_URL}/tab-illo.png`}
+                      alt="Carbon illustration"
+                    />
+                  </div>
+                </div>
+              </div>
+            </Tab>
+            <Tab {...props.tab} label="Design">
+              <div className="bx--grid bx--grid--no-gutter bx--grid--full-width">
+                <div className="bx--row landing-page__tab-content">
+                  <div className="bx--col-lg-16">
+                    Rapidly build beautiful and accessible experiences. The
+                    Carbon kit contains all resources you need to get started.
+                  </div>
+                </div>
+              </div>
+            </Tab>
+            <Tab {...props.tab} label="Develop">
+              <div className="bx--grid bx--grid--no-gutter bx--grid--full-width">
+                <div className="bx--row landing-page__tab-content">
+                  <div className="bx--col-lg-16">
+                    Carbon provides styles and components in Vanilla, React,
+                    Angular, and Vue for anyone building on the web.
+                  </div>
+                </div>
+              </div>
+            </Tab>
+          </Tabs>
+        </div>
+      </div>
+      <div className="bx--row landing-page__r3">
+        <div className="bx--row landing-page__r3">
```

Comment:
You have an extra `div` wrapper here

Reply:
I've removed it, and had to delete my branch to successfully push. See the referenced PR #1116